### PR TITLE
Scroll transaction

### PR DESCRIPTION
**Pull request by Powerade**

### CODE REVIEW

File: src/app/overview/container.tsx [hunk 1]
```diff
@@ -32,11 +30,14 @@ export default class Overview extends React.Component<Props & DispatchProps, any
   public componentDidMount(): void {
     this.props.getCurrentPriceInUSD()
     this.props.getCurrentPriceInBTC()
-    this.props.getTransactions()
   }
 
   public render(): any {
     const values: any = mapValues(this.props.overview, (value: string) => parseFloat(value).toFixed(MAX_DECIMALS))
+    const transactions: any = this.props.transactions.transactions ?
```
Comment (Powerade) [inline src/app/overview/container.tsx]: Can we create a `const transactions = this.props.transactions.transactions` ? This is too big

File: src/app/transactions/container.tsx [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+@connect(mapStateToProps, mapDispatchToProps)
+export default class Transactions extends React.Component<Props & DispatchProps, any> {
+  // tslint:disable-next-line:typedef
+  public componentDidMount() {
+    this.props.getTransactions()
+  }
 
-export default class Transactions extends React.Component {
   public render(): any {
+    const transactions: any = this.props.transactions.transactions || []
```
Comment (Powerade) [inline src/app/transactions/container.tsx]: Do we need this ` || []`? The default state on the reducer is the empty array already right?

File: src/app/overview/container.tsx (hunk 1 above)
Comment (Powerade) [reply src/app/overview/container.tsx]: Also, see below comment:
"Do we need this `|| []`? The default state on the reducer is the empty array already right?"

Comment (Hyjaz) [reply src/app/overview/container.tsx]: no it was returning an error

**Pull request merged**